### PR TITLE
[tests-only] pin core commit id

### PR DESCRIPTION
**Pull request by kiranparajuli589**
part of https://github.com/owncloud/web/issues/4885

### CODE REVIEW

**Comment by kiranparajuli589** [comment]
commitID actually works for the `git_reference` but in the `.star` file as we've to source the commit id from the .env file....imo its not possible to pin the commitID to the `git_reference` from outside using that docker container

File: .drone.star [hunk 1]
```diff
@@ -1046,7 +1048,7 @@ def installCore(version, db):
 		'image': 'owncloudci/core',
 		'pull': 'always',
 		'settings': {
-			'version': version,
+			'git_reference': commitID,
```
Comment (individual-it) [inline .drone.star:1051]: that does not work?
```suggestion
			'git_reference': $CORE_COMMITID,
```

Comment (kiranparajuli589) [reply .drone.star:1051]: no..because we have to  source the .env file which we can only do on steps part

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14906/

<p>

















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14919/

<p>


















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14920/

<p>

















</p></details>

**Review by individual-it — COMMENTED** [review]
the drone changes look good, but some tests fail

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14921/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14928/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14935/

<p>





</p></details>

**Review by phil-davis — COMMENTED** [review]
LGTM - please squash.

**Review by individual-it — APPROVED** [review]

**Pull request merged**